tests/bgp/test_bgp_authentication.py

```diff
@@ -145,7 +145,7 @@ def test_bgp_peer_group_password(setup):
 
     # mismatch peer group passwords
     cmd = 'vtysh -n {} -c "config" -c "router bgp {}" -c "neighbor {} password {}" -c "neighbor {} password {}" \
-          -c "end"'.format(setup['neigh_asic_index'], setup['dut_asn'], setup['peer_group_v4'], mismatch_pass,
+          -c "end"'.format(setup['asic_index'], setup['dut_asn'], setup['peer_group_v4'], mismatch_pass,
                            setup['peer_group_v6'], mismatch_pass)
 
     command_output = setup['duthost'].shell(cmd, module_ignore_errors=True)
```

Comment:
I believe original intention was to missmatch password on neighbor:
command_output =  setup['neighhost'].shell(cmd, module_ignore_errors=True)